### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by ziyue5**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (sandeep-ps) [inline CHANGELOG.md:7]: Please add the release version and date details here. Thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
...
+- Table styles [#76](https://github.com/policy-design-lab/pdl-frontend/issues/76)
+- Updated EQIP JSON data files. [#84](https://github.com/policy-design-lab/pdl-frontend/issues/84)
+
+### Fixed
+- Table sorting functions [#49](https://github.com/policy-design-lab/pdl-frontend/issues/49)
+- Dynamic api uri creation [#78](https://github.com/policy-design-lab/pdl-frontend/issues/78)
+- Tooltip styles for semi donut chart [#80](https://github.com/policy-design-lab/pdl-frontend/issues/80)
+- Number format shown on map when hovering [#82](https://github.com/policy-design-lab/pdl-frontend/issues/82)
+- Errors in the EQIP JSON data files. [#87](https://github.com/policy-design-lab/pdl-frontend/issues/87)
+
```
Comment (sandeep-ps) [inline CHANGELOG.md:30]: At the end of the CHANGELOG, if we add links for comparing adjacent releases, the section titles will become links to compare what changed between releases easily. This is in the Keep a Changelog format. 

Please see these for reference: 
https://raw.githubusercontent.com/policy-design-lab/pdl-api/main/CHANGELOG.md
https://github.com/policy-design-lab/pdl-api/blob/main/CHANGELOG.md

File: CHANGELOG.md (hunk 1 above)
Comment (sandeep-ps) [reply CHANGELOG.md:7]: You can use this for reference: https://github.com/policy-design-lab/pdl-api/blob/develop/CHANGELOG.md?plain=1

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (sandeep-ps) [inline CHANGELOG.md:7]:
```suggestion
## [0.3.0] - 2023-05-03
```

File: CHANGELOG.md (hunk 2 above)
Comment (sandeep-ps) [reply CHANGELOG.md:32]: Resolved in PR #106

File: CHANGELOG.md (hunk 1 above)
Comment (sandeep-ps) [reply CHANGELOG.md]: Resolved in PR #106

**Review by sandeep-ps — APPROVED** [review]
I tested this PR by running the application modules (both frontend and backend) locally and visiting a few pages. I couldn't find any issues. I will approve. Thanks.

**Review by navarroc — APPROVED** [review]

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**